models/mdex/bnb/mdex_bnb_trades.sql

```diff
@@ -0,0 +1,104 @@
+{{ config(
+    alias = 'trades'
+    ,partition_by = ['block_date']
+    ,materialized = 'incremental'
+    ,file_format = 'delta'
+    ,incremental_strategy = 'merge'
+    ,unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address']
+    ,post_hook='{{ expose_spells(\'["bnb"]\',
+                                      "project",
+                                      "sushiswap",
+                                    \'["codingsh"]\') }}'
+    )
+}}
+
+{% set project_start_date = '2021-04-05' %}
+
+WITH mdex_dex AS (
+    SELECT  t.evt_block_time                                             AS block_time,
+            `to`                                                         AS taker,
+            sender                                                       AS maker,
+            CASE WHEN amount0Out = 0 THEN amount1Out ELSE amount0Out END AS token_bought_amount_raw,
+            CASE WHEN amount0In = 0 THEN amount1In ELSE amount0In END    AS token_sold_amount_raw,
+            cast(NULL as double)                                         AS amount_usd,
+            CASE WHEN amount0Out = 0 THEN token1 ELSE token0 END         AS token_bought_address,
+            CASE WHEN amount0In = 0 THEN token1 ELSE token0 END          AS token_sold_address,
+            t.contract_address                                           AS project_contract_address,
+            t.evt_tx_hash                                                AS tx_hash,
+            ''                                                           AS trace_address,
+            t.evt_index
+    FROM {{ source('mdex_bnb', 'MdexPair_evt_Swap') }} t
+    INNER JOIN {{ source('mdex_bnb', 'MdexFactory_evt_PairCreated') }} f
```

Comment:
`mdex_bnb.MdexPair_evt_Swap` exists while `mdex_bnb.MdexFactory_evt_PairCreated` seems not available in v2 at the moment.

Reply:
MdexFactory its working now